mungegithub/mungers/flake-manager.go

```diff
@@ -41,7 +43,14 @@ const failedStr = "Failed: "
 var (
 	// pullRE is a regexp that will extract the PR# from a path to a flake
 	// that happened on a PR.
-	pullRE = regexp.MustCompile("pull/([0-9]+)/")
+	pullRE               = regexp.MustCompile("pull/([0-9]+)/")
+	failureInnerRE       = regexp.MustCompile("(?s)(.*)\n" + failedStr + "([^\n]*)\n\n```\n(.*)\n```\\s*$")
```

Comment:
Please comment saying what these are supposed to match. `gubernatorLinkRE` is clear enough, but why do we need the others?